sympy/integrals/__init__.py

```diff
@@ -9,6 +9,7 @@
     >>> integrate(sin(x),x)
     -cos(x)
 """
+from .deltafunctions import deltaintegrate
```

Comment:
Retrying with ```automethod``` -> ```autofunction```.
Not sure whether that's the right fix, but I have no way to test Sphinx stuff locally so at least I'll know whether the build fails or not.